lib/Sema/DerivedConformances.h

```diff
@@ -167,6 +167,18 @@ class DerivedConformance {
   /// \returns the derived member, which will also be added to the type.
   ValueDecl *deriveDecodable(ValueDecl *requirement);
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Derive a Parameterized requirement for a nominal type.
+  ///
+  /// \returns the derived member, which will also be added to the type.
+  ValueDecl *deriveParameterized(ValueDecl *requirement);
+
+  /// Derive a Parameterized type witness for a nominal type, if it has
+  /// parameters (stored properties marked with @parameter).
```

Comment:
`@TFParameter`

Reply:
Fixed in https://github.com/apple/swift/pull/18171/commits/99b20ffce93152d2e7d2b19d8e2371f1a601b1d3.